src/main/Extensions/Base64Conversion/Base64Conversion.ts

```diff
@@ -0,0 +1,87 @@
+import { AssetPathResolver } from "@Core/AssetPathResolver";
+import { Extension } from "@Core/Extension";
+import { Translator } from "@Core/Translator";
+import { SearchResultItem, SearchResultItemActionUtility } from "@common/Core";
+import { Image } from "@common/Core/Image";
+import { Buffer } from "buffer";
+import { InvocationArgument } from "./InvocationArgument";
```

Comment:
Please import types only when possible:

```suggestion
import type { AssetPathResolver } from "@Core/AssetPathResolver";
import type { Extension } from "@Core/Extension";
import type { Translator } from "@Core/Translator";
import { type SearchResultItem, SearchResultItemActionUtility } from "@common/Core";
import type { Image } from "@common/Core/Image";
import { Buffer } from "buffer";
import type { InvocationArgument } from "./InvocationArgument";
```